setup.py

```diff
@@ -92,12 +92,16 @@
             'tensorflow>=1.13.1,<2.0; python_version >= "3.7"',
         ],
         "bert": [
-            'transformers~=2.11.0; python_version>="3.6"',
-            'sentence-transformers~=0.2.6; python_version>="3.6"',
+            'transformers~=3.5.1; python_version>="3.6"',
+            'sentence-transformers~=0.3; python_version>="3.6"',
             "elasticsearch>=7.0",
         ],
         "examples": [
             'connexion>=2.7.0; python_version>="3.6"',
         ],
+        "augment":[
+            'torch~=1.7.0; python_version>="3.6"',
+            'transformers~=3.5.1; python_version>="3.6"'
```

Comment:
I think we may have to explicitly install SentencePiece library which the multilingual paraphraser needs.